src/services/dashboards/dashboard-customize/modules/DashboardWidgetInputForm.vue

```diff
@@ -300,12 +344,14 @@ export default defineComponent<Props>({
             } else {
                 // delete property schema
                 delete _widgetOptionsJsonSchema.properties[propertyName];
+                delete state.inheritItemMap[propertyName];
             }
             state.widgetOptionsJsonSchema = _widgetOptionsJsonSchema;
-            hideOptionsMenu();
+            hideContextMenu();
         };
         const handleClickAddOptions = () => {
-            state.addOptionsMenuVisible = true;
+            initiateMenu();
+            showContextMenu();
```

Comment:
suggestion: I think if you click when `visibleMenu` is true, shouldn't it be changed to false?

Reply:
I think its better, too. Thanks!